app/src/main/java/com/github/featuredetectandroid/ui/Menu.kt

```diff
@@ -0,0 +1,84 @@
+package com.github.featuredetectandroid.ui
+
+import android.content.SharedPreferences
+import android.util.Log
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment.Companion.Bottom
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.content.edit
+import com.github.featuredetectandroid.utils.stringToAlgorithmMap
+
+private const val TAG = "Menu"
```

Comment:
Personally I believe that UI shouldn't produce any logs -- its one of the business logic's prerogatives

Reply:
I deleted this logging call.

Reply:
Put it back into the function that does the change as it's now in `MainActivity` -- it's great to log such kind of things

Reply:
fixed